src/mixins.js

```diff
@@ -100,3 +100,11 @@ export const errors = {
     },
   },
 }
+
+export const isPlayButton = {
+  computed: {
+    isPlayButton() {
+      return !this.$envConfig.webtritRecordingDisablePlayButton
+    },
```

Comment:
After renaming env variable this method will be `isRecordingPlayButtonDisable` and with such name `!` must beware removed during return.